src/Filament/Resources/AbstractContentResource.php

```diff
@@ -373,6 +377,29 @@ public static function getVanityURLFields(): array
 
     }
 
+    public static function getRoleRestrictionFields(): array
+    {
+        $roleRestrictionFields = [
+            Section::make('Allowed Roles')
+                ->compact()
+                ->description('List of roles that are allowed to access this content. If no role is selected, the content is hidden to non-authenticated viewers.')
```

Comment:
We need to change this logic so that if no role is selected, it's available to everyone (including users who aren't logged in).

The way this is currently, there's no way to make content just viewable to all users, including guests?

Reply:
It's not a required field, so without a role attached, it should work on Guests.

Reply:
But your info text says "If no role is selected (e.g, it's blank) the content is hidden to non-authenticated viewers"?

Reply:
Ohh sorry that should be is a role is seelcted